Data xform from attachment

Mahesh-Krishnan1 opened the pull request:
New BR called Transform Data from Attachment added to help non-admins load data into SN

earlduque commented:
@Mahesh-Krishnan1 I deleted an irrelevant file for you and moved your readme file to the `Business Rules` folder, you had it in `Business Rule`

Mahesh-Krishnan1 commented:
Thank you! I know I created it first under the wrong folder but then deleted it and added both to the correct folder; must have not done something right.


­­­[signature_1955581459]<https://wipro.com/>
Mahesh Krishnan
Americas 2 – iDEAS Apps & Data
T: +1.480.524.9512


From: Earl Duque ***@***.***>
Date: Thursday, October 20, 2022 at 12:02 PM
To: ServiceNowDevProgram/code-snippets ***@***.***>
Cc: Mahesh Krishnan ***@***.***>, Mention ***@***.***>
Subject: Re: [ServiceNowDevProgram/code-snippets] Data xform from attachment (PR #543)

CAUTION:This email is received from an external domain. Open the hyperlink(s) & attachment(s) with caution.
.


@Mahesh-Krishnan1<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FMahesh-Krishnan1&data=05%7C01%7Cmahesh.krishnan%40wipro.com%7Cbf840d9bd22b460536cc08dab2cdae6b%7C258ac4e4146a411e9dc879a9e12fd6da%7C1%7C0%7C638018893761779374%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=b6RcayFuDCnLVtFverbd1Z%2FqEJE5x1YaYjeHpKmIllE%3D&reserved=0> I deleted an irrelevant file for you and moved your readme file to the Business Rules folder, you had it in Business Rule

—
Reply to this email directly, view it on GitHub<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FServiceNowDevProgram%2Fcode-snippets%2Fpull%2F543%23issuecomment-1286007524&data=05%7C01%7Cmahesh.krishnan%40wipro.com%7Cbf840d9bd22b460536cc08dab2cdae6b%7C258ac4e4146a411e9dc879a9e12fd6da%7C1%7C0%7C638018893761779374%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=e3%2Bq8Bi9BT%2FIgBTjQMzlAY%2BLf6yIFOu7z6d2diUEPeY%3D&reserved=0>, or unsubscribe<https://apc01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAVHL5PEBLBMG2XAHZB3LX7TWEGJNNANCNFSM6AAAAAARIQ6UHI&data=05%7C01%7Cmahesh.krishnan%40wipro.com%7Cbf840d9bd22b460536cc08dab2cdae6b%7C258ac4e4146a411e9dc879a9e12fd6da%7C1%7C0%7C638018893761779374%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=Vit5i6wv65n%2FvlShTfS6uhiAXSm9CqtfpxEBrPigoeE%3D&reserved=0>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

'The information contained in this electronic message and any attachments to this message are intended for the exclusive use of the addressee(s) and may contain proprietary, confidential or privileged information. If you are not the intended recipient, you should not disseminate, distribute or copy this e-mail. Please notify the sender immediately and destroy all copies of this message and any attachments. WARNING: Computer viruses can be transmitted via email. The recipient should check this email and any attachments for the presence of viruses. The company accepts no liability for any damage caused by any virus transmitted by this email. www.wipro.com'

Internal to Wipro
